src/devices/Common/Iot/Device/Graphics/IGraphics.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Iot.Device.Graphics
+{
+    /// <summary>
+    /// This declarative interface provides the basis for drawing functions using image specific APIs.
+    /// This interface is empty. The implementation is provided via extension methods.
+    /// </summary>
+    public interface IGraphics : IDisposable
```

Comment:
why not just regular class in that case? what's the value of this being interface and having extension methods elsewhere?

Reply:
Doesn't really matter whether it's an interface or an empty class type. In either case, it's intended to be a handle only. The reason I'm using extension methods is because it is up to the implementation to provide the set of drawing functions. With this approach, it is possible to add new drawing functions without changing the interface.